lib/cartopy/io/img_tiles.py

```diff
@@ -497,6 +497,47 @@ def find_images(self, target_domain, target_z, start_tile=None):
                 yield self.tms_to_quadkey(tile, google=True)
 
 
+class OrdnanceSurvey(GoogleWTS):
+    # https://developer.ordnancesurvey.co.uk/os-api-framework-agreement
```

Comment:
Please add a class docstring, or it will get the `GoogleWTS` one, which doesn't make too much sense here.